src/app/modules/auth/__tests__/auth.routes.spec.ts

```diff
@@ -38,8 +38,8 @@ describe('auth.routes', () => {
 
       // Assert
       expect(response.status).toEqual(400)
-      expect(response.body).toEqual({
-        message: 'Some required parameters are missing',
+      expect(response.body).toMatchObject({
+        message: 'celebrate request validation failed',
```

Comment:
We can update these assertions to be more specific, including the full error shape that is returned to the frontend.
i.e continue using `toEqual` :P